src/sentry/options/defaults.py

```diff
@@ -994,6 +994,13 @@
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+# An option to enable writing from the new schema for the caching indexer
+register(
+    "sentry-metrics.indexer.write-new-cache-namespace",
+    default=False,
+    flags=FLAG_AUTOMATOR_MODIFIABLE,
+)
```

Comment:
Is this option going to be shared between the release health pipeline and the generic metrics pipeline? If yes, would it be better to have different options for both pipelines and reduce blast radius?

Reply:
> Would it be better to have different options for both pipelines and reduce blast radius?

I am less inclined to do so, since this toggles on an additional write step, (i.e. once we flip this to `True`, the indexer will do an extra write to the new namespace). It doesn't change the way we read. I think having 2 would just increase the mental load, since we can turn it off anytime we like